### ISSUE TITLE
Organic Chemistry - In Exercises tool, <figure>...</figure> does not output numbered figure

### ISSUE THREAD

**Issue by ww-openstax** [openstax/cnx-recipes#5096]
# Brief Description of Scenario
In Exercises tool, `<figure>...</figure>`  does not output numbered figure

Screenshot
![image](https://user-images.githubusercontent.com/89537088/230661779-75dd4697-7237-4c6f-acc6-ad3de475f218.png)

# Acceptance Criteria
- [ ] figure tagging in injected exercises also get baked
- [ ] no disruption for the general figure numbering 
  
# Sample Mark-Up
```
<figure><img class="scaled-down" src="https://exercises.openstax.org/rails/active_storage/blobs/proxy/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBZ0VVIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--41d4527dc370aea3df21f8daa23e9302189ec9f0/OChem_12_02_006b.jpg" alt="An illustration shows two mass spectra. ... a peak is shown at 30 percent abundance.">
<figcaption>Mass spectra for Problem 12-2.</figcaption>
</figure>
```

# Location(s)
- **osbooks repo:** osbooks-organic-chemistry
- **collection slug:** organic-chemistry
- **ref/branch:**
- **chapter/section:** Chapter 12 / Problems 12-2 and 12-9 // Nickname:12-02-OC-P02 // Nickname:12-08-OC-P09
- **surrounding text:**

**Comment by lwtchu** [comment]
@wilkus27 Interesting - so if I put an ID, it should number 🤔 ?

**Comment by lwtchu** [comment]
![Screenshot 2023-04-14 at 3 05 29 PM](https://user-images.githubusercontent.com/11299125/232144683-b3218a95-a985-4224-a76c-217931f5da33.png)
@ww-openstax I've been able to get the number to show up by adding an ID to the figure tags. I used the ID format from POET but added a 2 so it doesn't conflict with anything that exists in the module. `fig-00201` and `fig-00202` on the exercise with Nickname:12-02-OC-P02.
